### DISCUSSION TITLE
Why does my instance warn me about license expiry?

### DISCUSSION THREAD

**Discussion by danya02** [goauthentik/authentik#8851] [Q&A] [answered]
I've installed Authentik about two weeks ago using [the Kubernetes helmchart](https://docs.goauthentik.io/docs/installation/kubernetes). I marked `akadmin` and my user as "internal". Now, I'm getting a banner that says `Warning: The current user count has exceeded the configured licenses`, with a link to the Enterprise config section

I didn't add any licenses, and I thought when installing it that I'm getting the non-enterprise version.

I've also changed both users to be called "external" instead, but the banner still remains:

![image](https://github.com/goauthentik/authentik/assets/15921166/0a35b217-2f89-4460-899e-a2a2b1710ef3)

According to https://docs.goauthentik.io/docs/enterprise/manage-enterprise#license-violation-notifications, seeing this warning means that in 4 more weeks the instance will become read-only. Should I be worried?

**Comment by rissson** [comment, marked as answer]
We fixed this in 2024.2.2. I recommend you upgrade to that version.
